Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -129,6 +127,28 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
             }
         }
     }
+
```

Comment:
Why can't we make the locationview simply the right height and instead of doing something relatively low level like `hitTest` instead just install the `tapGestureRecognizer` on the whole `BrowserLocationView` instead of the `locationLabel` ? Then the whole thing from top to bottom is a hit target. Teh label needs to have its `userIntractionAllowed` removed then so that clicks pass through it.